GitDepend.IntegrationTests/Scenarios/HelpPageScenario.cs

```diff
@@ -30,21 +57,47 @@ public void InvalidVerb_Returns_InvalidCommand()
         [Test]
         public void CloneVerb_ShouldCloneAllRepos()
```

Comment:
This is all in the HelpPageScenario, which should focus only on Help page work flows. You should create a new Scenario file for Cloning.